Description: Fix segmentation fault when pressing g in full screen mode
--- a/xpdf/XPDFViewer.cc
+++ b/xpdf/XPDFViewer.cc
@@ -839,6 +839,9 @@
 
 void XPDFViewer::cmdFocusToPageNum(GString *args[], int nArgs,
 				   XEvent *event) {
+  if (core->getFullScreen()) {
+    return;
+  }
   XmTextFieldSetSelection(pageNumText, 0,
 			  strlen(XmTextFieldGetString(pageNumText)),
 			  XtLastTimestampProcessed(display));
